examples/data_collect/main.cpp

```diff
@@ -693,18 +672,7 @@ int main(int argc, char *argv[]) {
 
         // We have both 8bit and 16bit pixels, compute the size in 8bit
         if (sensorName == "adsd3500") {
-            if (mode == 7) {
-                //one 16bit depth, one 16bit ab, one 8bit confidence
-                // =>> 5 * 8 bit subframes
-                subFrames = 5;
-            } else if (mode == 10) {
-                // 4 x 16 bit subframes =>> 8 x 8 bit subframes
-                subFrames = 8;
-            } else if (fDetails.type == "vga") {
-                //one 16bit depth, one 16bit ab, one 8bit confidence
-                // =>> 5 * 8 bit subframes
-                subFrames = 5;
-            }
+            subFrames = 5;
```

Comment:
lr-native needs 8 subframes and sr-native needs 6. The rest are 5.

Reply:
ok.